tests/test_image_driver.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python 
+''' VIC Image Driver testing '''
+
+import xarray as xr
+import numpy as np 
+import numpy.testing as npt 
+
+def assert_nan_equal(ds_domain, ds_output): 
+    """ 
+    test to see if nans in two data arrays are the same in order to check domain 
+    first dataarray is domain file, second dataarray is a variable in the output
+    dataset for image driver 
+    """
+
+    # get lat/lng dim names from domain file 
+    match_vars = ds_domain.coords
```

Comment:
You actually get more than the `dim` names here, you get the coordinates. You should add this distinction to your comment.

I would also change this line slightly to make sure we only check the coordinates that are in the `mask` variable:

`match_vars = ds_domain['mask'].coords`